content/doc/book/pipeline/syntax.adoc

```diff
@@ -105,16 +105,20 @@ section is placed. The section must be defined at the top-level inside the
 [[differences-between-top-and-stage-level]]
 ===== Differences between top and stage level Agents
 
-There are some nuances when adding an agent to the top level or a stage level, and this when the `options` directive is applied.
+There are some nuances when adding an agent to the top level or a stage level, and this when the `option` directive is applied.
+Check the section
+link:../syntax/#options[options]
+for more information.
```

Comment:
```suggestion
There are some nuances when adding an agent to a top level or stage level, and this is when the `option` directive is applied.
Refer to the link:../syntax/#options[options] section for more information.
```
suggestion to have one sentence per line, as this makes it easier for reviewers to read, and adjusting a bit of wording to read correctly